### PR TITLE
Update DemoModule.py

### PR DESCRIPTION
**Pull request by rafalnowak**

### CODE REVIEW

File: DemoModule.py [hunk 1]
```diff
@@ -5,6 +5,12 @@ def nice_function(value):
     return value**2
```
Comment (rafalnowak) [inline DemoModule.py]: Nice change

File: DemoModule.py [hunk 2]
```diff
@@ -5,6 +5,12 @@ def nice_function(value):
...
 
 
+def bad_code(bar):
+    foo = bar*2
+    xxx = 50
```
Comment (sputnikci-test) [inline DemoModule.py]: asfsaf

File: DemoModule.py (hunk 1 above)
Comment (sputnikci-test) [inline DemoModule.py]: Nice change

Comment (sputnikci-test) [inline DemoModule.py]: Nice change

Comment (sputnikci-test) [inline DemoModule.py]: Nice change

Comment (sputnikci-test) [inline DemoModule.py]: Nice change

**Pull request closed without merging**